src/components/analytics-toggle/AnalyticsToggle.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import Checkbox from '../checkbox/Checkbox'
+import Details from '../details/Details'
+
+const AnalyticsToggle = ({ doToggleAnalytics, analyticsEnabled, t }) => {
+  // Simplify fetching a list of i18n keys.
+  const items = Array(9).fill(1)
+  return (
+    <React.Fragment>
+      <Checkbox className='dib bg-white pa3' onChange={doToggleAnalytics} checked={analyticsEnabled} label={
```

Comment:
```suggestion
      <Checkbox className='dib pa3' onChange={doToggleAnalytics} checked={analyticsEnabled} label={
```

I prefer this without the white background to blend with the rest of the Settings, but I'll leave it up to you!